src/rocky/index.js

```diff
@@ -0,0 +1,37 @@
+var rocky = require('rocky');
+
+var dataLogs = [];
+
+// Triggered when memory pressure is an issue
+rocky.on('memorypressure', function(event) {
+  switch(event.level) {
+    case 'high':
+      // Memory pressure is high, free some memory
+      dataLogs = [];
+      console.log('relieved the pressure');
+      break;
+    case 'normal':
+    case 'low':
+      // Not implemented
```

Comment:
could you still log this? maybe as the first line of the handler? `console.log('memory pressure event', event.level);` or something?